documentation/howto/maven_basics.adoc

```diff
@@ -0,0 +1,172 @@
+.. Set the value of the `Packaging` field to `pom`.
+.. (Optional) Add a name for your project in the `Name` field.
+.. (Optional) Add a description for your project in the `Description` field.
++
+.Configure Project Details
+image::mvnbasic_maven_project.png[Configure Project Details]
++
+.. Click `Finish` to conclude the new Maven project creation wizard.
+
+*Result:* Your new Maven project is created and appears in the `Project Explorer` view.
+
+[[New_Module]]
+== 2. Create a New Maven Module
+
+Each Maven project with a packaging pom can include multiple Maven modules. Follow the instructions to create your first Maven module:
+
+=== Prerequisites
+. You must have an existing Maven project available with the packaging type *pom*. See <<New_Project,Create a New Maven Project>> for instructions to create a new Maven project.
+
+=== Creating a New Maven Module
+. Create a new Maven module as follows:
+.. In your workspace, navigate to menu:File[New > Other].
+.. From the wizard selection screen, expand `Maven` and select `Maven Module`.
+.. Click `Next >` to continue.
+. In the first step of the `New Maven Module` wizard, enter the initial module details:
+.. Select the `Create a simple project (skip archetype selection)` check box. If this check box is selected, the `Select an Archetype` step of the wizard is skipped.
+.. Enter the desired module name in the `Module Name` field. This value corresponds to the Maven project's Project ID.
+.. Use the `Browse` button to locate the desired parent project and select it.
+.. The following optional steps are available for further configuration, if required:
+... (Optional) Select the `Add project(s) to working set` check box to add the newly created projects to a working set.
+... (Optional) Click `Advanced` to view additional optional advanced configuration for the new Maven project, such as:
+.... `Resolve Workspace projects`: dependencies opened as workspace projects will be resolved without having to install them to your local Maven repository first. This way, any changes made to one of these dependencies will have an immediate effect on other projects consuming it (compilation, refactoring, etc.). When `Resolve Workspace projects` is disabled, dependencies existing in the workspace must be installed to your local Maven repository after any change (by running `mvn install`), in order to see effects in projects consuming them.
+.... `Profiles`: select a set of Maven profiles to activate or deactivate in the workspace. Profiles are defined in the project `pom.xml`, or inherited from a parent `pom.xml`, or defined in the relevant `settings.xml`.
+.... `Name templates`: allows you to disambiguate projects names in the workspace by prepending or appending the group ID or SCM branch names to the default artifact ID.
++
+.Set the Module Name and Parent
+image::mvnbasic_module_name_parent.png[Set the Module Name and Parent]
++
+.. When the configuration is complete, click `Next` to continue.
```

Comment:
For this module, we want to describe how to use the archetype selection. We're not trying to create another pom project